src/test/ui/rfc-2008-non-exhaustive/enum_same_crate_empty_match.rs

```diff
@@ -25,7 +25,7 @@ pub enum EmptyNonExhaustiveEnum {}
 fn empty_non_exhaustive(x: EmptyNonExhaustiveEnum) {
     match x {}
     match x {
-        _ => {} // not detected as unreachable
+        _ => {}, //~ ERROR unreachable pattern
```

Comment:
In theory, this is a breaking change. Though I view this as fixing a bug, if we are to fix it, we'll need to phase it out: first by emitting warnings that it will break, then eventually converting them to hard errors.

Reply:
It's a warn-by-default lint though. It's only an error in that test because it has `deny(unreachable_patterns)`

Reply:
Ah, of course.